charts/stable/audiobookshelf/values.yaml

```
@@ -0,0 +1,49 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: advplyr/audiobookshelf
+  # -- image tag
+  tag: 1.6.69
```

```suggestion
  # -- image tag
  # @default -- chart.appVersion
  tag:
```
We are migrating to a pattern where we leave the `tag` field empty in our values.yaml. This makes it so we only have to maintain the version under the `appVersion` field in `Chart.yaml` since that is what is used by default when no value is specified.

---

that's good to know!  Is that a helm feature where that is passed in automatically?

---

Not necessarily a generic Helm feature, but we default to `chart.appVersion` for the image tag in our library when none is specified.